dashboard/client/src/components/CommonRecentCard.tsx

```diff
@@ -0,0 +1,144 @@
+import { createStyles, makeStyles, Typography } from "@material-ui/core";
+import classNames from "classnames";
+import _ from "lodash";
+import React, { ReactNode } from "react";
+import { Link } from "react-router-dom";
+import {
+  LinkWithArrow,
+  OverviewCard,
+} from "../pages/overview/cards/OverviewCard";
+
+type CommonRecentCardProps = {
+  headerTitle: string;
+  className: string | undefined;
```

Comment:
instead of adding this manually, you can do `CommonRecentCardProps = { ...} & ClassNameProps`.

Just import ClassNameProps